ivy/functional/ivy/experimental/layers.py

```diff
@@ -770,3 +770,65 @@ def embedding(
         else:
             ret[i] = weights[x, :]
     return ret
+
+
+@handle_exceptions
+@to_native_arrays_and_back
+@handle_array_like
```

Comment:
This decorator has been renamed to `@handle_array_like_without_promotion`. Could you kindly make this change as currently all the CI tests are failing due to this error